src/test/app/Path_test.cpp

```diff
@@ -560,20 +672,43 @@ class Path_test : public beast::unit_test::suite
         env.fund(XRP(10000), "alice", "bob", "carol", "dan", gw, gw2);
         env.close();
         env(rate("carol", 1.1));
+        env.close();
```

Comment:
I don't think you have to close after each of these calls. Just close after the last `pay`.

Reply:
yup we only need to close ledger after payments. Had spent a lot of time debugging this test because it did not close after each Payment 😂